lib/linguist/languages.yml

```diff
@@ -1577,7 +1577,7 @@ HCL:
   ace_mode: ruby
   codemirror_mode: ruby
   codemirror_mime_type: text/x-ruby
-  tm_scope: source.ruby
+  tm_scope: source.terraform
```

Comment:
[Terraform](https://terraform.io) is the software (an infrastructure automation tool, written by a company called Hashicorp). I called the Sublime package Terraform because I thought that would be easier for people to find than HCL (Hashicorp Configuration Language). I think it's best to keep the source name consistent with the package name, and if I rename the package in the future then I'll rename the source to match?